examples/static-hbase-logs/emr.tf

```diff
@@ -0,0 +1,78 @@
+locals {
+  this_application = ["Hbase"]
+}
+
+# Create new EC2 key pair
+resource "tls_private_key" "emr_private_key" {
+  algorithm = "RSA"
+}
+
+module "emr_key_pair" {
+  source     = "terraform-aws-modules/key-pair/aws"
+  version    = "1.0.0"
+  key_name   = "hbase-test-emr-key"
+  public_key = tls_private_key.emr_private_key.public_key_openssh
+  tags       = var.tags
+}
+
+# EMR Static HBase cluster
+module "emr-hbase" {
+  # source = "git::git@github.com:Datatamer/terraform-aws-emr.git?ref=6.2.0"
```

Comment:
You are going to need a dependency with the VPC Endpoint in order for this not to fail randomly. EMR takes longer to boot than Tamr-VM but i'd rather be safe than sorry.